packages/components/src/components/list-item-location/readme.md

```diff
@@ -86,6 +86,7 @@ A `listItemDidRender` event is emitted from the `<mi-list-item-location>` elemen
 | `icon`           | `icon`             |                                  | `string`                                   | `undefined` |
 | `iconBadge`      | `icon-badge`       |                                  | `string`                                   | `undefined` |
 | `iconBadgeValue` | `icon-badge-value` |                                  | `string`                                   | `undefined` |
+| `level`          | `level`            |                                  | `string`                                   | `'Level'`   |
```

Comment:
Can we just use `Level` without the '' ?

Reply:
It signifies that the default value is the string `'Level'`.

Reply:
Alright 😄